src/components/CurrencySelector/CurrencyOptions.tsx

```diff
@@ -1,35 +1,16 @@
 import { Combobox } from "@headlessui/react";
-import { useCurrencisQuery } from "services/aws/wise";
+import { Currency } from "./types";
 import { isEmpty } from "helpers";
 
 type Props = {
   classes?: string;
-  searchText?: string;
+  currencies: Currency[];
 };
 
-export default function Options({ classes = "", searchText = "" }: Props) {
-  const { data: currencies = [] } = useCurrencisQuery(
```

Comment:
Loading currencies inside `CurrencySelector` makes it impossible to have different sources of currency data - this becomes a problem when we want to load Stripe-supported currencies, as opposed to all Wise currencies.

By allowing the parent to load currencies from whichever source they need and pass them down, we address this problem and make `CurrencySelector` more flexible.